addons/binding/org.openhab.binding.dwdunwetter/README.md

```diff
@@ -0,0 +1,114 @@
+The warnings will be sorted by severity and for warnings with same severity by the valid from date.
+This ensures that in the channels for the first warning will always be the highest severity.
+If the API returns more warnings than the configured number in the thing, the warnings with the lowest severity will be dropped.  
+ 
+| Channel      | Type            | Description                                                                      |
+|--------------|-----------------|----------------------------------------------------------------------------------|
+| warningN     | Switch          | ON if a warning is present, OFF else                                             |
+| UpdatedN     | Trigger Channel | Triggers NEW if a warning is send the first time                                 |
+| severityN    | String          | Severity of the warning. Possible values are Minor, Moderate, Severe and Extreme |
+| headlineN    | String          | Headline of the warning like "Amtliche Warnung vor FROST"                        |
+| descriptionN | String          | Textual description of the warning                                               |
+| eventN       | String          | Type of the warning, e.g. FROST                                                  |
+| effectiveN   | DateTime        | Issued Date and Time                                                             |
+| onsetN       | DateTime        | Start Date and Time for which the warning is valid                               |
+| expiresN     | DateTime        | End Date and Time for which the warning is valid                                 |
+| altitudeN    | Number:Length   | Lower Height above sea level for which the warning is valid                      |
+| ceilingN     | Number:Length   | Upper Height above sea level for which the warning is valid                      |
+| urgencyN     | String          | Urgency of the warning Possible values are Future and Immediate                  |
+| instructionN | String          | Additional instructions and safety informations                                  |
+
+All channels are readonly!
+
+The main purpose of the channel _warning_ is to be used for controlling visibility in sitemaps.
+Or the check in rules if there is warning present. 
+It should not be used for rules that need to fire if a new warning shows up. 
+If a warning is replaced by another warning, that channel stays at ON, there will be no state change. 
+For rules the need to fire if a new warning show up there is the trigger channel _updatedN_. 
+That trigger channel fires an event whenever a warning is send the first time to that thing.
+That channel triggers if a warning is replaced by another.
+
+More explanations about the specific values of the channels can be found in the documentation of the DWD at: [CAP DWD Profile 1.2](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_dwd_profile_de_pdf.pdf?__blob=publicationFile&v=7 "CAP DWD Profile 1.2")   
```

Comment:
The URL part behind the space seems unnecessary?

```suggestion
More explanations about the specific values of the channels can be found in the documentation of the DWD at: [CAP DWD Profile 1.2](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_dwd_profile_de_pdf.pdf?__blob=publicationFile&v=7)
```

Reply:
Done